CONTRIBUTING.md

```diff
@@ -1,17 +1,16 @@
 # Contribute
 
 ### Step 1
--  Fork the repository
+-  Fork the repository.
```

Comment:
If we're going to do this, should probably do it for all steps, personally don't think we need the full stop.